helm_deploy/laa-estimate-eligibility/values/production.yaml

```diff
@@ -49,7 +49,7 @@ googleTagManager:
   containerId: GTM-WJ98FR4
 
 cfe:
-  host: http://check-financial-eligibility-partner-check-financial-partner.check-financial-eligibility-partner-production.svc.cluster.local
+  host: http://cfe-civil.cfe-civil-production.svc.cluster.local
```

Comment:
Could we add a temporary commit to add this value to uat.yaml as well so we can verify before deploy that this hostname is correct? (Last week the config was updated so that production CFE is at least accessible from the CCQ UAT namespace - c.f. https://github.com/ministryofjustice/cloud-platform-environments/pull/12082)